test/sanity/pep8/README.md

```diff
@@ -24,3 +24,10 @@ Files listed in the [skip list](skip.txt) are not tested by `pep8`.
 ## Removed Files
 
 Files which have been removed from the repository must be removed from the legacy file list and the skip list.
+
+## Running locally
+
+You can do a quick pep8 spotcheck by approximating `command_sanity_pep8` in `test/runner/lib/executor.py` like this:
+
+    ./test/runner/ansible-test sanity -v --test pep8 <files to sanity-check>
```

Comment:
I'd revise the example a little. Verbose output is probably not needed in most cases:

```
test/runner/ansible-test sanity --test pep8 [file-or-directory-path-to-check] ...
```